### PR TITLE
Support numeric value in Ikea tz air_purifier_fan_mode.

### PR DESCRIPTION
**Pull request by hakaneriksson**

### CODE REVIEW

**Comment by Koenkk** [comment]
@sjorge could you review this?

File: devices/ikea.js [hunk 1]
```diff
@@ -147,21 +147,24 @@ const ikea = {
...
+                switch (value) {
                 case 'off':
                     fanMode = 0;
                     break;
                 case 'auto':
                     fanMode = 1;
                     break;
                 default:
-                    fanMode = parseInt(((parseInt(value) / 2.0) * 10) + 5);
+                    fanMode = (+value / 0.2) + 5;
```
Comment (sjorge) [inline devices/ikea.js:163]: The +value here is very confusing to read, can this go back to parseInt?

**Review by sjorge — CHANGES_REQUESTED** [review]
The rest looks fine, it was originally string because the enum for ha was sending strings, accepting both is probably better

File: devices/ikea.js (hunk 1 above)
Comment (hakaneriksson) [reply devices/ikea.js]: Sure, reverted the unary plus operator. For me it got an error when setting fan mode from HA.

**Review by sjorge — APPROVED** [review]
LGTM

**Comment by Koenkk** [comment]
Thanks!

**Pull request merged**